### PR TITLE
Fixes # 963 Added space for drag bar

### LINKED ISSUE #963: UI - can't move window by bar
On windows 10 can't move client window by grabbing it by top bar 
top_cant_move_only_green_small_line_is_ok

### PR DESCRIPTION
**Pull request by johnhce**
Added space for drag bar at top of main window.
Fixes #963

### CODE REVIEW

File: src/main/resources/theme/main.fxml [hunk 1]
```diff
@@ -20,6 +20,10 @@
         <VBox maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" minHeight="0.0" minWidth="0.0"
               VBox.vgrow="ALWAYS">
             <children>
+                <HBox alignment="CENTER_LEFT" maxHeight="-Infinity"
+                      maxWidth="1.7976931348623157E308" minHeight="18" minWidth="-Infinity">
+	                <Pane maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" HBox.hgrow="ALWAYS"/>
+	            </HBox>
```
Comment (1-alex98) [inline src/main/resources/theme/main.fxml]: Please format correctly

**Review by 1-alex98 — CHANGES_REQUESTED** [review]

**Review by 1-alex98 — APPROVED** [review]

File: src/main/resources/theme/main.fxml (hunk 1 above)
Comment (1-alex98) [reply src/main/resources/theme/main.fxml]: Use Intelij and turn on auto formatting ;)

**Comment by 1-alex98** [comment]
Please dont use a fork but make branches in the main project so we can checkout and test more easily

**Comment by micheljung** [comment]
Why add two additional nodes with a hardcoded size when the same (read: better) could be achieved using simple CSS?

What's even the purpose of the nested `Pane` node? Or the HBox around it?

**Comment by 1-alex98** [comment]
only minheight is hardcoded tho

**Comment by 1-alex98** [comment]
You want a padding at the top?

**Comment by micheljung** [comment]
Plus, the commit message says "Fixes # 963" which, due to the space after the hash, does not close #963.
Plus, my comment in https://github.com/FAForever/downlords-faf-client/issues/963 seems to have been ignored, so now there's also spacing when the window is maximized, which is bad for usability.

Please revert this, and please pay more attention to quality (I'm not even mentioning the 7 ghost-commits in this PR).

**Comment by 1-alex98** [comment]
Reverted

**Comment by 1-alex98** [comment]
I squased the commits but also told him 1 issue = 1 commit

**Pull request merged**